central/metadata/service/service_impl.go

```diff
@@ -51,9 +51,10 @@ func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName strin
 // GetMetadata returns the metadata for Rox.
 func (s *serviceImpl) GetMetadata(ctx context.Context, _ *v1.Empty) (*v1.Metadata, error) {
 	metadata := &v1.Metadata{
-		BuildFlavor:   buildinfo.BuildFlavor,
-		ReleaseBuild:  buildinfo.ReleaseBuild,
-		LicenseStatus: v1.Metadata_VALID,
+		BuildFlavor:      buildinfo.BuildFlavor,
+		ReleaseBuild:     buildinfo.ReleaseBuild,
+		LicenseStatus:    v1.Metadata_VALID,
+		IsManagedService: env.ManagedCentral.BooleanSetting(),
```

Comment:
I think, it should be fairly trivial to create a unit test covering the new attribute. Do you mind adding one?

Also, there's https://github.com/stackrox/stackrox/blob/0d3fe397cbc35475c8057e5abae608996e25adf5/tests/metadata_test.go but I don't know if there's a way to distinguish the case when the attribute is absent from the case when the attribute is set to `false`.